components/skeleton/demo/skeleton-group-wrapper.js

```diff
@@ -0,0 +1,12 @@
+import '../../collapsible-panel/collapsible-panel.js';
+import '../../collapsible-panel/collapsible-panel-group.js';
+import { html, LitElement } from 'lit';
+import { SkeletonGroupMixin } from '../skeleton-group-mixin.js';
+
+class SkeletonGroupTestWrapper extends SkeletonGroupMixin(LitElement) {
+	render() {
+		return html`<slot></slot>`;
+	}
+}
+customElements.define('d2l-skeleton-group-test-wrapper', SkeletonGroupTestWrapper);
```

Comment:
Yeah, I'm not too familiar with the test repo yet, so I'm not entirely sure where it'll go but happy to move it wherever.

Reply:
I think Stacey meant to say "directory" instead of "repo". Like this file is just used in visual-diffs, but it's located in the "demo" directory and likely should be in the "test" directory with the visual-diffs. You could even just move it to be an inline `<script type="module">` in the visual-diff HTML files if you like.

Reply:
ohhh, I see! Yeah might go the inline approach then -- I didn't think that was a possibility 🤯.

Reply:
Yup sorry about the confusion, definitely meant to say test _folder_, not repo! 🤦 Got testing repos on the brain